ddtrace/internal/datadog/profiling/util.py

```diff
@@ -1,12 +1,23 @@
 from sys import version_info
 from typing import Any  # noqa:F401
 
+from ddtrace.internal.compat import ensure_binary
 from ddtrace.internal.logger import get_logger
 
+from .types import StringType
+
 
 LOG = get_logger(__name__)
 
 
+def ensure_binary_or_empty(s: StringType) -> bytes:
+    try:
+        return ensure_binary(s)
+    except Exception:
+        pass
+    return b""
```

Comment:
ci complaining

https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/64558/workflows/6c9df835-5137-48b5-823a-1cb055e4be10/jobs/3989355

>> Issue: [B110:try_except_pass] Try, Except, Pass detected.
   Severity: Low   Confidence: High
   CWE: CWE-703 (https://cwe.mitre.org/data/definitions/703.html)
   More Info: https://bandit.readthedocs.io/en/1.7.5/plugins/b110_try_except_pass.html
   Location: ddtrace/internal/datadog/profiling/util.py:16:4
15	        return ensure_binary(s)
16	    except Exception:
17	        pass
18	    return b""

Reply:
Oops, I'm not running the security lints locally--thank you